tests/cli/test_rasa_train.py

```diff
@@ -38,6 +39,54 @@ def test_train(run_in_default_project):
     assert os.path.basename(files[0]) == "test-model.tar.gz"
 
 
+def test_train_core_compare(run_in_default_project):
+    temp_dir = os.getcwd()
+
+    write_yaml_file(
+        {
+            "language": "en",
+            "pipeline": "supervised_embeddings",
+            "policies": [{"name": "KerasPolicy"}],
+        },
+        "config_1.yml",
+    )
+
+    write_yaml_file(
+        {
+            "language": "en",
+            "pipeline": "supervised_embeddings",
+            "policies": [{"name": "MemoizationPolicy"}],
+        },
+        "config_2.yml",
+    )
+
+    run_in_default_project(
+        "train",
+        "core",
+        "-c",
+        "config_1.yml",
+        "config_2.yml",
+        "--stories",
+        "data/stories.md",
+        "--out",
+        "core_comparison_results",
+        "--runs",
+        "2",
+        "--percentages",
+        "25",
+        "75",
+        "--augmentation",
+        "5",
+    )
+
+    assert os.path.exists(os.path.join(temp_dir, "core_comparison_results"))
+    sub_dirs = list_subdirectories(os.path.join(temp_dir, "core_comparison_results"))
```

Comment:
maybe we can make the naming a bit clearer so that the test is easier understandable
```suggestion
    run_directories = list_subdirectories(os.path.join(temp_dir, "core_comparison_results"))
```